integrations/pinecone/tests/test_document_store.py

```diff
@@ -257,3 +260,59 @@ def test_embedding_retrieval(self, document_store: PineconeDocumentStore):
         assert len(results) == 2
         assert results[0].content == "Most similar document"
         assert results[1].content == "2nd best document"
+
+    @pytest.mark.skipif("PINECONE_API_KEY" not in os.environ, reason="PINECONE_API_KEY not set")
```

Comment:
```suggestion
```
this marker is inherited from `TestDocumentStore` test class, so we don't need it.